environment.ubuntu.yml

```diff
@@ -1,230 +1,14 @@
-  - xorg-libxrender=0.9.10=h516909a_1002
-  - xorg-renderproto=0.11.1=h14c3975_1002
-  - xorg-xextproto=7.3.0=h14c3975_1002
-  - xorg-xproto=7.0.31=h14c3975_1007
-  - xz=5.2.4=h14c3975_1001
-  - zlib=1.2.11=h14c3975_1004
-  - freeimageplus=3.18.0=hf484d3e_2
-  - gl2ps=1.3.8=h14c3975_4
-  - oce=0.17.2=h6bb024c_10
-  - pythonocc-core=0.17=py27_1
-  - mkl=2017.0.3=0
-  - mkl-service=1.1.2=py27_3
-  - dbus=1.13.2=h714fa37_1
-  - gst-plugins-base=1.14.0=hbbd80ab_1
-  - gstreamer=1.14.0=hb453b48_1
-  - libgcc-ng=8.2.0=hdf63c60_1
-  - libgfortran-ng=7.3.0=hdf63c60_0
-  - libstdcxx-ng=8.2.0=hdf63c60_1
-  - pcre=8.43=he6710b0_0
-  - qt=5.6.3=h8bf5577_3
-  - pip:
-    - alabaster==0.7.12
-    - babel==2.6.0
-    - backports.shutil-get-terminal-size==1.0.0
-    - bleach==3.1.0
-    - chardet==3.0.4
-    - cloudpickle==0.8.1
-    - configparser==3.7.4
-    - cvxopt==1.2.3
-    - decorator==4.4.0
-    - defusedxml==0.5.0
-    - docutils==0.14
-    - doit==0.29.0
-    - entrypoints==0.3
-    - ephem==3.7.6.0
-    - funcsigs==1.0.2
-    - h5py==2.9.0
-    - idna==2.8
-    - imagesize==1.1.0
-    - importlib-resources==1.0.2
-    - ipaddress==1.0.22
-    - ipykernel==4.10.0
-    - ipython==5.8.0
-    - ipython-genutils==0.2.0
-    - ipywidgets==7.4.2
-    - jinja2==2.10.1
-    - joblib==0.12.5
-    - jsonschema==3.0.1
-    - jupyter==1.0.0
-    - jupyter-client==5.2.4
-    - jupyter-console==5.2.0
-    - jupyter-core==4.4.0
-    - llvmlite==0.28.0
-    - lxml==4.3.3
-    - markupsafe==1.1.1
-    - mistune==0.8.4
-    - mock==2.0.0
-    - nbconvert==5.4.1
-    - nbformat==4.4.0
-    - networkx==2.2
-    - notebook==5.7.8
-    - numba==0.43.1
-    - packaging==19.0
-    - pandocfilters==1.4.2
-    - pathlib2==2.3.3
-    - patsy==0.5.1
-    - pbr==5.1.3
-    - pexpect==4.6.0
-    - pickleshare==0.7.5
-    - pkginfo==1.5.0.1
-    - plotly==3.7.1
-    - prometheus-client==0.6.0
-    - prompt-toolkit==1.0.15
-    - ptyprocess==0.6.0
-    - py4design==0.27
-    - pycollada==0.6
-    - pygments==2.3.1
-    - pyinotify==0.9.6
-    - pymc3==3.6
-    - pymf==0.1.9
-    - pyrsistent==0.14.11
-    - pysal==1.14.4.post2
-    - pyshp==2.1.0
-    - pyyaml==5.1
-    - pyzmq==18.0.1
-    - qtconsole==4.4.3
-    - readme-renderer==24.0
-    - requests==2.21.0
-    - requests-toolbelt==0.9.1
-    - retrying==1.3.3
-    - salib==1.2
-    - scandir==1.10.0
-    - scikit-learn==0.20.3
-    - seaborn==0.9.0
-    - send2trash==1.5.0
-    - simplegeneric==0.8.1
-    - snowballstemmer==1.2.1
-    - sphinx==1.8.5
-    - sphinxcontrib-websupport==1.1.0
-    - terminado==0.8.2
-    - testpath==0.4.2
-    - theano==1.0.4
-    - timezonefinder==4.0.1
-    - tqdm==4.31.1
-    - traitlets==4.3.2
-    - twine==1.13.0
-    - typing==3.6.6
-    - urllib3==1.24.1
-    - utm==0.4.2
-    - wcwidth==0.1.7
-    - webencodings==0.5.1
-    - widgetsnbextension==3.4.2
-    - xlrd==1.2.0
-    - xlwt==1.3.0
-    - deap==1.3.0
+  - geopandas
+  - numba=0.42.0
+  - python=2.7
+  - osmnx
```

Comment:
We could remove `osmx` from the conda env file since it is already in **INSTALL_REQUIRES** of `setup.py` as I am seeing this when running `pip install` after `conda create` :
```
  Found existing installation: osmnx 0.11
    Uninstalling osmnx-0.11:
      Successfully uninstalled osmnx-0.11

Successfully installed osmnx-0.9 
```

Just a minor issue, I will test this and change it if needed